### ISSUE TITLE
Lock Service Not work perfact

### ISSUE THREAD

**Issue by solankiranvir** [twinone/AppLocker#44]
1. 1st time suppose i lock Google Play Store app okay.When i open play store app it show okay it works.
After that Suppose i stop service from left navigation drawer & Play Store already lock now when i open play store app still it show lock screen even if service is stop.2nd thing now i open app lock app so in that in navigation drawer it display that app lock service start though i already stopped it last time okay so 1st issue is this.
2. If i remove lock from play store app from list click okay & service is start.Now when i open play store app it show lock screen though i already remove lock from this last time okay this is 2nd issue.
SO how can i solve this 2 issue.

**Comment by twinone** [comment]
Please check if this issue is present in [this apk](https://www.twinone.org/AppLocker_44.apk)

**Comment by solankiranvir** [comment]
okay i will check & give u feedback.

**Comment by solankiranvir** [comment]
okay it work but still issue in this suppose i stop service from left drawer and when i back to app it show that service is start.

**Comment by solankiranvir** [comment]
Understand issue?

**Comment by solankiranvir** [comment]
Hello can i know what changes u have done in last apk which you send me for testing?

**Comment by twinone** [comment]
I had the same issue, but I thought I fixed it.

Is your part 1. now fixed in that apk?

**Comment by solankiranvir** [comment]
Yes part 1 is fixed in last apk so can you send me that perticular change code? after i also try for 2nd one.

**Comment by twinone** [comment]
Sorry, I wrote part 1, but i meant part 2.
What is actually fixed? Updating the service with the correct apps when you lock/unlock them? (You don't need to restart the service when you lock/unlock an app)

**Comment by solankiranvir** [comment]
ya sorry in last apk part 2 solved i only need what you changed for solve that issue?so i only change in that file not again import whole project.

**Comment by twinone** [comment]
Well, please wait until I fix the other part and push the changes. Then you can pull them.

What are you using the project for? Please read the notice I have put in the [README](https://github.com/twinone/AppLocker#copying) about copying.

**Comment by solankiranvir** [comment]
I already read all document don't worry i use your project for learning purpose & you think why this person give more interest in this.the reason is i use latest design concept also from project.

**Comment by solankiranvir** [comment]
Okay i will wait

**Comment by solankiranvir** [comment]
Hello any update of issue 44-1?

**Comment by twinone** [comment]
Please be patient. I have other things to do too. If you need it fast, you
can implement the solution yourself and send me a pull request.

**Comment by solankiranvir** [comment]
I just ask not that give me solution i ask because i also try to solve this issue thats why.

**Comment by twinone** [comment]
If you want, you can. I have a lot of things to do this week, sorry.

I just ask not that give me solution i ask because i also try to solve this
issue thats why.

—
Reply to this email directly or view it on GitHub
<https://github.com/twinone/AppLocker/issues/44#issuecomment-133371545>.

**Comment by solankiranvir** [comment]
Okay

**Comment by solankiranvir** [comment]
Can You give me part 44-2 solution in which file you change ?

**Comment by twinone** [comment]
Well, it was already fixed in master, I just sent the compiled APK because I suspected it was already fixed.

**Comment by solankiranvir** [comment]
This solution is in #41 & #42 issue right?
